test/suites/erc20.ts

```diff
@@ -89,10 +89,10 @@ export default (context: TestContext) => {
         }),
       );
   };
-
```

Comment:
Feels like you're a Go programmer and you hate blank lines.

Reply:
I don't hate 

blank lines. But I am a Go developer. I'm honestly not sure why or how this one got deleted.